pkgs/tools/inputmethods/fcitx5/fcitx5-lua.nix

```
@@ -1,10 +1,11 @@
-{ lib, stdenv
+{ pkgs
+, lib
+, stdenv
 , fetchFromGitHub
 , cmake
 , extra-cmake-modules
 , fcitx5
-, lua5_3
-, luaPackage ? lua5_3
+, luaPackage ? pkgs.lua5_3
```

```suggestion
, lua5_3
```

And add

```
let
  lua = lua5_3
in
```

Because lua5_3 can work with callPackage and we can use lua in the future when nixpkgs use lua5_3 as default.